js/views/itemEditVw.js

```diff
@@ -143,7 +143,7 @@ module.exports = Backbone.View.extend({
 
   changeType: function(e) {
     var typeValue = $(e.target).val();
-    if(typeValue === "physical good") {
+    if(typeValue === "physical good") { //TODO: remove magic text, make constant or lookup
```

Comment:
The only time this value would change is if the API also changed. I don't think putting it in a config file would provide a benefit. 

In general, the user model acts as a config file, and settings like the server URL and the SSL flag are (or will be) stored there.